src/moves/FreeScroll.ts

```diff
@@ -56,12 +58,12 @@ class FreeScroll extends Snap {
   }
 
   public findPanelWhenInterrupted(ctx: MoveTypeContext): DestinationInfo {
-    const { viewport } = ctx;
+    const { viewport, axesEvent } = ctx;
 
     return {
       panel: viewport.getNearestPanel()!,
       destPos: viewport.getCameraPosition(),
-      duration: viewport.options.duration,
```

Comment:
This duration can't be more than 0 as it's just stopping on camera position. It's my fault.
Could you please fix that for me?

Reply:
@WoodNeck It was a fix.